### PR TITLE
added cat sprite

### PR DESCRIPTION
**Pull request by anshuman-nandan**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜

## Breaking changes
- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Screenshots
- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

File: src/art/anshuman2/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document2</title>
```
Comment (sophiabrandt) [inline src/art/anshuman2/index.html:8]: Please provide a more descriptive title.

File: src/art/anshuman2/script.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const charcter = document.getElementById('cter');
+const charcontainer = charcter.getContext('2d');
+const charwid = charcter.width = 623;
+const charhit = charcter.height = 572;
+const charimg = new Image();
+charimg.src = "images/sprite.png";
```
Comment (sophiabrandt) [inline src/art/anshuman2/script.js:6]: Please provide the license to the image you used, so that we can be sure that we don't violate copyright

**Comment by sophiabrandt** [comment]
Please fill out the request template.

**Comment by MattCSmith** [comment]
Closing due to unresolved change request. Feel free to resolve and reopen.

**Pull request closed without merging**